ptf

```diff
@@ -66,6 +66,7 @@ config_default = {
     "platform_args"      : None,
     "platform_dir"       : os.path.join(root_dir, "platforms"),
     "interfaces"         : [],
+    "device_type"        : "bm",
```

Comment:
Same remark as above, should be done with this: https://github.com/p4lang/ptf#passing-parameters-to-tests